src/models/model.cpp

```diff
@@ -204,6 +204,12 @@ std::string Tokenizer::Decode(std::span<const int32_t> tokens) const {
   return string;
 }
 
+std::string Tokenizer::ApplyChatTemplate(const char* template_str, const char* messages, bool add_generation_prompt) const {
+  const char* output;
+  CheckResult(OrtxApplyChatTemplate(tokenizer_, template_str, messages, output, add_generation_prompt, false));  // bool tokenize = false
```

Comment:
What is the commented out bool tokenize for?

Reply:
oh it's not a commented out parameter, it was to clarify what the last parameter was for (false) , but I can remove it if it's unclear, or change the comment to state that more explicitly.